instrumentation/apache-camel-2.20/javaagent-unittests/src/test/groovy/io/opentelemetry/javaagent/instrumentation/apachecamel/MessageAttributeNamesFactoryTest.groovy

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apachecamel
+
+import io.opentelemetry.api.DefaultOpenTelemetry
+import io.opentelemetry.api.GlobalOpenTelemetry
+import io.opentelemetry.api.baggage.propagation.W3CBaggagePropagator
+import io.opentelemetry.context.propagation.ContextPropagators
+import io.opentelemetry.extension.trace.propagation.B3Propagator
+import org.apache.camel.component.aws.sqs.SqsConfiguration
+import org.apache.camel.component.aws.sqs.SqsEndpoint
+import spock.lang.Specification
+
+class MessageAttributeNamesFactoryTest extends Specification {
+
+  def "should Add Single Field To Empty Names List"() {
+
+    setup:
+    // baggage = single field
+    GlobalOpenTelemetry.set(
+      DefaultOpenTelemetry.builder()
```

Comment:
Why use `DefaultOpenTelemetry` up here, and the global in the next test?

Reply:
whoops...misread. Is the goal with this to have a propagator-only implementation installed?